index.bs

```diff
@@ -257,7 +257,8 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 <h4 id="trace" oldids="trace-data,dom-console-trace" method for="console">trace(...<var>data</var>)</h4>
 
-Perform Logger("error", <var>data</var>).
+Performs Logger("log", <var>data</var>) and prints a snapshot of the call-stack from where the method was called.
```

Comment:
As far as I can tell implementations don't seem to log the passed _data_. Chrome uses it as a sort of label, but others just ignore it. Also, it would be good to use Printer() for the printing. Here I would say something like:

1. Let _trace_ be an implementation-specific, potentially-interactive representation of the callstack from where this method was called. Optionally, incorporate _data_ as a label for this representation.
1. Perform Printer("log", _trace_).

And then modify Printer to talk about console.trace() in addition to %o and %O.